tests/features/field.multilingual.feature

```diff
@@ -6,92 +6,78 @@ Feature: Field Multilingual features
-	And I press the "Filter" button
-	And I click "edit"
-	And I fill in "edit-translations-fr" with "Mon Groupe Couleur"
-	And I press the "Save translations" button
-	Then I should see the success message "The string has been saved."
-
-	@api
-	Scenario Outline: Check translation of field and field group labels
-	# fix missing translation (deleted because languages are deleted at the end of scenario)
-	Given I run drush "sqlq" '"INSERT INTO locales_target SELECT lid , 0x4d6f6e2047726f75706520436f756c657572, \\"fr\\",  0, 0, 0 FROM locales_source WHERE source = \\"My Group Color\\""'
-	Given I am an anonymous user
-	When I go to "<url>"
-	Then I should see "<title>" in the "#page-title" element
-	And I should see "<grouplabel>" in the "fieldset.group-groupcolor span.fieldset-legend" element
-	And I should see "<fieldlabel>" in the "div.field-name-field-selectcolor div.field-label" element
-	And I should see "<fieldvalue>" in the "div.field-name-field-selectcolor div.field-items div.field-item" element
-
-	Examples:
-      | url                                       | title             | grouplabel         | fieldlabel               | fieldvalue |
-      | content/title-english_en                  | Title in English  | My Group Color     | Select a Color           | Red        |
-      | content/title-english_fr                  | Titre en Français | Mon Groupe Couleur | Selectionner une Couleur | Rouge      |
-      | content/title-english_de                  | Title in English  | My Group Color     | Select a Color           | Red        |
-      | content/title-english_en?2nd-language=fr  | Title in English  | My Group Color     | Select a Color           | Red        |
-      | content/title-english_de?2nd-language=fr  | Titre en Français | Mon Groupe Couleur | Selectionner une Couleur | Rouge      |
 
-	Scenario: Administrator can delete field group labels
-	# Finally, delete the added fields
-	Given I am logged in as a user with the 'administrator' role
-	When I go to "admin/structure/types/manage/page/fields/field_selectcolor/delete"
-	And I press the "Delete" button
-	Then I should see the success message "The field Select a Color has been deleted from the Basic page content type."
-	When I go to "admin/structure/types/manage/page/groups/group_groupcolor/delete/default"
-	And I press the "Delete" button
-	Then I should see the success message "The group My Group Color has been deleted from the Basic page content type."
+  Scenario: Administrator can translate field and field group labels. Once it is done,
+  users see field and field group labels translated in interface language or in the source language
+  if translations are not done
+    Given I am logged in as a user with the 'administrator' role
+    # Add and translate field and field Group
+    And a field with the following settings is added to "page" type:
+      | Label           | Select a Color  |
+      | Name            | selectcolor     |
```

Comment:
The name doesn't have the traditional syntax. In this case, it should be "field_select_a_color‎".